apps/showcase/tsconfig.app.json

```diff
@@ -35,6 +35,7 @@
     "preserveWatchOutput": true,
     "jsx": "react",
     "allowSyntheticDefaultImports": true,
+    "types": ["node"],
```

Comment:
why `node` in the tsconfig used to parse the app code?
Should not be a good idea.
Do you wanted to put it in the tsconfig dedicate to test?

Reply:
Actually the `"types": ["node"],` were present in the root testing `tsconfig`. Only the `@types/node` were missing from the `package.json` here